doc/REST-interface.md

```diff
@@ -71,6 +71,7 @@ Responds with 404 if the block doesn't exist.
 `GET /rest/blockhashbyheight/<HEIGHT>.<bin|hex|json>`
 
 Given a height: returns hash of block in best-block-chain at height provided.
+Responds with 404 if block height is out of range.
```

Comment:
```suggestion
Responds with 404 if block not found
```

Reply:
Agreed

Reply:
This suggestion improves the statement and goes along with the classical use of **error 404**.